Fixes:
wvConfig.c:3608:19: error: incompatible function pointer types assigning to 'startElementSAXFunc'
wvConfig.c:3609:17: error: incompatible function pointer types assigning to 'endElementSAXFunc'
--- wvConfig.c.orig	2006-10-25 15:52:18
+++ wvConfig.c	2024-11-09 22:44:17
@@ -157,7 +157,7 @@
 
 
 static void
-exstartElement (void *userData, const char *name, const char **atts)
+exstartElement (void *userData, const unsigned char *name, const unsigned char **atts)
 {
     unsigned int token_type;
     expand_data *mydata = (expand_data *) userData;
@@ -3370,7 +3370,7 @@
 }
 
 static void
-exendElement (void *userData, const char *name)
+exendElement (void *userData, const unsigned char *name)
 {
     /*
        expand_data *mydata = (expand_data *)userData;
